Description:
  - This patch concerns the manpage.
  - Updates the homepage.
  - Removes deprecated option (-na|--no-alltray).
  - Adds new options: --key, --notray, -- nominimize and --borderless.
  - Corrects the alltray version number.
Author: Ignace Mouzannar <mouzannar@gmail.com>
Last-Update: 2010-02-07
--- a/alltray.1
+++ b/alltray.1
@@ -17,7 +17,7 @@
 .el .ne 3
 .IP "\\$1" \\$2
 ..
-.TH ALLTRAY 1 "april 26, 2006" "Version 0.66" "AllTray Manual Page"
+.TH ALLTRAY 1 "2010-02-07" "Version 0.70" "AllTray Manual Page"
 .SH NAME
 AllTray \- Dock any program into the system tray.
 .SH "SYNOPSIS"
@@ -71,8 +71,8 @@
 \fB\-stask\fR \fB\-\-skip-taskbar\fR
 not visible in taskbar\&.
 .TP
-\fB\-na\fR \fB\-\-no-alltray\fR
-no "(Alltray)" in window title\&.
+\fB\-x\fR \fB\-\-borderless\fR
+remove border, title, frame (if not supported native)\&.
 .TP
 \fB\-m\fR \fB\-\-menu\fR
 Add entry "menu text:command" to popdown menu\&.
@@ -83,14 +83,29 @@
 \fB\-g\fR \fB\-\-geometry\fR
 initial position. see man X.
 .TP
+\fB\-k\fR \fB\-\-key\fR [Modifier:]Key
+create a keyboard shortcut.
+.br
+Modifier \-> "Shift", "Control", "Alt", "AltGr"
+.br
+Key      \-> "a", "F1", "End" ...
+.br
+or "Keycode" (Number) returned by the program "xev"
+.TP
+\fB\-nt\fR \fB\-\-notray\fR
+display no tray icon (useful only with the "\-\-key" option).
+.TP
+\fB\-nm\fR \fB\-\-nominimize\fR
+click on window close button: do not minimize back to system tray, close.
+.TP
 \fB\-conf\fR \fB\-\-configure\fR
 show KDE configuration dialog\&.
 
 
 .SH HOMEPAGE
-http://alltray.sourceforge.net
-
-.SH "AUTHOR"
+http://alltray.trausch.us/
+
+.SH "AUTHORS"
 
 .PP
 This manual page was written by Carlos C Soto <csoto@sia\-solutions\&.com> for \
@@ -98,4 +113,4 @@
 distribute and/or modify this document under the terms of the GNU General Public License, \
 Version 2 any later version published by the Free Software Foundation\&.
 .PP
-AllTray (C) 2005, Jochen Baier <email@jochen-baier.de>
+Copyright 2005-2008, Jochen Baier <email@jochen-baier.de>
